lib/charms/velero_libs/v0/velero_backup_config.py

```diff
@@ -177,6 +182,8 @@ def __init__(
             app_name (str): The name of the application for which the backup is configured
             relation_name (str): The name of the relation. (from metadata.yaml)
             spec (VeleroBackupSpec): The backup specification to be used
+            refresh_event (Optional[Union[BoundEvent, List[BoundEvent]]]):
+                Optional event(s) to trigger data sending.
```

Comment:
Let's add the `refresh_event` option in the example and explain that is used to update the relation databag